internal/access/credential.go

```diff
@@ -23,11 +24,26 @@ func CreateCredential(c *gin.Context, user models.Identity) (string, error) {
 		return "", HandleAuthErr(err, "user", "create", models.InfraAdminRole)
 	}
 
-	tmpPassword, err := generate.CryptoRandom(12, generate.CharsetPassword)
+	settings, err := data.GetSettings(db)
+	if err != nil {
+		return "", err
+	}
+
+GENERATE:
+	tmpPassword, err := generate.CryptoRandom(settings.LengthMin, generate.CharsetPassword)
```

Comment:
we can't use the min length as the tmp password length. what if the min is 4? it should be the `max(LengthMin, InfraDefaultSize)`